text_generation/causal_lm/cpp/continuous_batching/library/CMakeLists.txt

```diff
@@ -17,6 +17,18 @@ FetchContent_MakeAvailable(nlohmann_json)
 
 find_package(OpenVINO REQUIRED COMPONENTS Runtime)
 
+function(ov_genai_build_jinja2cpp)
+    FetchContent_Declare(jinja2cpp
+	    jinja2cpp
+    GIT_REPOSITORY https://github.com/jinja2cpp/Jinja2Cpp.git
```

Comment:
let's use TAR.GZ instead of git, because it's faster and git is not required.

See `pybind11` in `python` folder as an example